src/Public/js/myradio.timeslot.js

```diff
@@ -69,12 +69,24 @@ $("#timeslots").on(
                     check.attr("name", "signin[]")
                       .attr("id", "signin_"+data[row].user.memberid)
                       .attr("value", data[row].user.memberid);
-                    label.attr("for", "signin_"+data[row].user.memberid)
-                      .html(data[row].user.fname + " " + data[row].user.sname);
+                    if (empty(data[row].user.nname) != false) {
+                      label.attr("for", "signin_"+data[row].user.memberid)
+                        .html(data[row].user.fname + ' "' + data[row].user.nname  + '" ' + data[row].user.sname);
```

Comment:
given how long this string concatenation has gotten, it should probably be broken out into a helper function

Reply:
This is true, this probably a case of upsetting something in some way. I'll split it off into a separate function for along with the other fixes